cscs-checks/tools/profiling_and_debugging/perftools.py

```diff
@@ -50,10 +50,6 @@ def __init__(self, lang):
                 'LIB=-lstdc++']
 
         self.executable = 'jacobi'
-        # NOTE: Reduce time limit because for PrgEnv-pgi even if the output
-        # is correct, the batch job uses all the time.
-        self.time_limit = (0, 5, 0)
```

Comment:
Are we letting the job run for 10 min? or the issue was resolved?

Reply:
The job finishes in time. This comment is no more valid, that's why I removed it.